bVNC/src/main/java/com/iiordanov/bVNC/dialogs/RateOrShareFragment.kt

```diff
@@ -60,10 +62,53 @@ class RateOrShareFragment : DialogFragment() {
         }
         versionAndCode = v.findViewById<View>(R.id.versionAndCode) as TextView
         versionAndCode?.text = Utils.getVersionAndCode(v.context)
-
+        setVisibilityOfElements(v)
         return v
     }
 
+    fun setTitle() {
+        if (Utils.isCustom(context)){
+            try {
+                dialog?.setTitle(
+                    getString(
+                        requireContext().resources.getIdentifier(
+                            Utils.getStringConfigAttribute(App.configFileReader.configData, TAG.replaceFirstChar { it.lowercase() }, "title", "key"),
+                            "string",
+                            Utils.pName(context)
+                        )
+                    )
+                )
+                return
+            }
+            catch (e: Exception) {
+                isCustomException(e)
+            }
+        }
+        dialog?.setTitle(getString(R.string.action_rate_or_share_app))
+    }
+
+    fun setVisibilityOfElements(v: View) {
+        if (Utils.isCustom(context)){
+            try {
+                Utils.setVisibilityForViewElementsViaConfig(context, App.configFileReader.configData, TAG.replaceFirstChar { it.lowercase() }, v)
+            }
+            catch (e: Exception) {
+                isCustomException(e)
+            }
+        }
+    }
+
+    fun isCustomException(e: Exception) {
+        when (e) {
+            is NullPointerException -> {
```

Comment:
To complement the previous comment - basically you've caught any exception and then have logic to check whether it's an NPE and re-throw if it's not. It's significantly simpler to just catch only the NPE instead.